modules/end-user-portal/src/Models/Response/OrganisationResult.cs

```diff
@@ -0,0 +1,48 @@
+using Newtonsoft.Json;
+
+namespace GpConnect.NationalDataSharingPortal.EndUserPortal.Models.Response;
+
+public class OrganisationResult
+{
+    [JsonProperty("id")]
+    public string OdsCode { get; set; } = "";
+
+    [JsonProperty("name")]
+    public string Name { get; set; } = "";
+
+    [JsonProperty("address")]
+    public OrganisationAddress Address { get; set; }
+}
+
+public class OrganisationAddress
+{
+    [JsonProperty("line")]
+    public List<string> AddressLines { get; set; }
+
+    [JsonProperty("city")]
+    public string City { get; set; } = "";
+
+    [JsonProperty("district")]
+    public string County { get; set; } = "";
+
+    [JsonProperty("postalCode")]
+    public string Postcode { get; set; } = "";
+
+    [JsonProperty("country")]
+    public string Country { get; set; } = "";
+
+    public string FullAddress => GetFullAddressAsString();
+
+    private string GetFullAddressAsString()
+    {
+        var addressLines = new List<string> {
```

Comment:
This looks to have double checking for not null, is this the result of a merge gone wrong?

Reply:
I think so - I've sorted it anyway.